src/plugins/crypto/README.md

```diff
@@ -148,23 +150,18 @@ Shutdown is enabled in the unit tests to prevent memory leaks.
 
 ## Examples ##
 
-### Metadata based encyption ###
-
-You specify the parameters of the cryptographic operations in a KeySet together with the keys to be encrypted.
-The following parameters are required:
-
-- **Key** - the symmetric cryptographic key for encryption
-- **IV** - the initialization vector (IV) that is required by the CBC mode
+To mount a backend with the gcrypt plugin variant that uses the GPG key 859E2AD7, use:
 
-The following keys are required for metadata based encryption:
+	kdb mount test.ecf user/t crypto_gcrypt "crypto/key=859E2AD7"
```

Comment:
Shouldn't we discourage the usage of short IDs? (I think you know the story about the faked keys that collide with keys from Linux Kernel Devs. I know it does not really apply here, because the keys need to be in the ring already, but it might be better from "educational" point of view, at least in the examples.)